packages/zowe-explorer-api/src/profiles/UserSettings.ts

```diff
@@ -9,6 +9,16 @@
  *
  */
 
+import * as zowe from "@zowe/cli";
+
+export interface DataSetAllocTemplate extends zowe.ICreateDataSetOptions {
+    /**
+     * Name of data creation template
+     */
+    name?: string;
+}
+export type dsAlloc = DataSetAllocTemplate;
```

Comment:
I think it would be helpful to define this type alias in pascal case to maintain our current convention (e.g., `DataSetAllocTemplate`) and so that it stands out to developers as a type when referenced.